src/sentencepiece_trainer.cc

```diff
@@ -153,6 +153,12 @@ util::Status SentencePieceTrainer::MergeSpecsFromArgs(
       CHECK_OR_RETURN(absl::SimpleAtoi(value, &v));
       absl::SetFlag(&FLAGS_minloglevel, v);
       continue;
+    } else if(key == "encode_unicode_case") {
+      bool encode_unicode_case;
+      std::istringstream(value) >> std::boolalpha >> encode_unicode_case;
```

Comment:
I would have thought there is a better way of doing that, like std::stoi but with bool, but not really. So this is OK.